src/main/java/com/neprozorro/rest/dto/LotInfoCriteriaRequestDto.java

```diff
@@ -1,23 +1,42 @@
 package com.neprozorro.rest.dto;
 
 import com.neprozorro.model.LotStatus;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Builder;
 import lombok.Getter;
 import lombok.Setter;
-import lombok.Builder;
 
 import java.util.List;
 
 @Getter
 @Setter
 @Builder
+@Schema(
+        title = "criteria of lot information",
+        description = "criteria witch uses in filter"
+)
 public class LotInfoCriteriaRequestDto {
 
+    @Schema(example = "Олекс")
     private String buyer;
+
+    @Schema(example = "Викт")
     private String seller;
+
     private List<LotStatus> lotStatus;
+
+    @Schema(example = "009023")
     private String dk;
+
+    @Schema(example = "10000,20000")
```

Comment:
if there is between, more or less can we add all possible variations of value we can add as a filter?